feat(service-worker): Support "Request"-Action (#46889)
angular/angular: aio/content/guide/service-worker-notifications.md

| `openWindow`                | Opens a new tab at the specified URL, which is resolved relative to the service worker scope.                                                                                                            |
| `focusLastFocusedOrOpen`    | Focuses the last focused client. If there is no client open, then it opens a new tab at the specified URL, which is resolved relative to the service worker scope.                                       |
| `navigateLastFocusedOrOpen` | Focuses the last focused client and navigates it to the specified URL, which is resolved relative to the service worker scope. If there is no client open, then it opens a new tab at the specified URL. |

> | `fetch` | Do nothing but sending a simple Http-GET to the specified URL. |

Reviewer: Nit:

```suggestion
| `fetch`                     | Send a simple GET request to the specified URL.                                                                                                                                                          |
```
